internal/build/container_test.go

```diff
@@ -33,7 +33,7 @@ ADD dir/c.txt .
 	f.WriteFile("dir/c.txt", "c")
 	f.WriteFile("missing.txt", "missing")
 
-	ref, err := f.b.BuildDockerfile(f.ctx, f.ps, f.getNameFromTest(), df, f.Path(), model.EmptyMatcher, model.DockerBuildArgs{})
+	ref, err := f.b.BuildDockerfile(f.ctx, f.ps, f.getNameFromTest(), df, f.Path(), model.EmptyMatcher, model.DockerBuildArgs{}, false)
```

Comment:
have you considered having a separate Tag() call?

i.e., docker tag fortune:tilt-cabcab fortune:latest

(i also don't think TagImage should be part of the normal BuildDockerfile call, but we can make that decision independently)